<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>414</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>414</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>414</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Invador</string>
  </property>
  <widget class="Gameboard" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>630</width>
     <height>414</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>630</width>
     <height>414</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,10">
    <item row="5" column="0">
     <widget class="QPushButton" name="btn_new_game">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&amp;New Game</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="8">
     <widget class="Judgement" name="battle_widget" native="true">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="btn_quit">
      <property name="text">
       <string>&amp;Quit</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLCDNumber" name="lcdScore">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn_shoot">
      <property name="text">
       <string>&amp;Shoot</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn_left">
      <property name="text">
       <string>&amp;Left</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btn_right">
      <property name="text">
       <string>&amp;Right</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="btn_help">
      <property name="text">
       <string>&amp;Help</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Gameboard</class>
   <extends>QWidget</extends>
   <header location="global">gameboard.h</header>
   <container>1</container>
   <slots>
    <slot>fire()</slot>
    <slot>hit()</slot>
    <slot>missed()</slot>
    <slot>newGame()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>Judgement</class>
   <extends>QWidget</extends>
   <header location="global">judgement.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
